### ISSUE TITLE
Doesn't work when exports is reassigned

### ISSUE THREAD

**Issue by emmanuelbuah** [sormy/webpack-fix-default-import-plugin#2]
This is probably just an FYI to note in the how-it-works section. If module.exports is manual overwritten, then this package will not work. Example, the url-loader produces modules akin to 

```
/* 492 */
/*!*******************************************!*\
  !*** ./src/assets/images/some-image.svg ***!
  \*******************************************/
/*! dynamic exports provided */
/*! all exports used */
/***/ (function(module, exports) {
module.exports = "data:image/svg+xml;base64,PHN2Z...."
/***/ }),
```

Because module.exports is reassigned instead of the default, patching it with  the code below doesn't work.
`` module.exports.default = module.exports ``

**Comment by sormy** [comment]
Could you please try this fix for your case? I will update library if it works for you.

```javascript
var fixModuleExports = function (module) {
    ...
    if (typeof module.exports === 'string') {
      module.exports = new String(module.exports);
    } else if (typeof module.exports === 'number') {
      module.exports = new Number(module.exports);
    } else if (typeof module.exports === 'boolean') {
      module.exports = new Boolean(module.exports);
    }
    module.exports.default = module.exports;
  }
}
```

**Comment by sormy** [comment]
Please use `esModuleInterop` option that is available since TypeScript 2.7 to use default imports in the same way as in Babel.
